README.md

```diff
@@ -136,6 +136,8 @@ contribute to Swarm development, you can build from source.
 1. Go get a snack while `stack` downloads a Haskell compiler and
    all of Swarm's dependencies.
 
+NOTE: To run tests, please use the [`scripts/run-tests.sh` script](scripts/run-tests.sh) to avoid problems with incremental compilation in alternation with `stack build`.
```

Comment:
This sounds too generic.

Besides, random contributors do not need to worry about it.

It would make much more sense as a TIP for regular developers in DEVELOPING.md to speed up Stack until it solves its issue.

Reply:
> in DEVELOPING.md

Sorry, which file is that?  I only see the one in the [VS Code plugin development directory](https://github.com/swarm-game/swarm/blob/main/editors/vscode/DEVELOPING.md).

Reply:
My mistake, this was the section I am thinking about which already has some tips:
https://github.com/swarm-game/swarm/blob/main/CONTRIBUTING.md#development-workflow

Reply:
Got it, fixed.